blogposts/2021/release-3.34.md

```diff
@@ -0,0 +1,45 @@
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.34/sourcegraph-3-34-release.png
+changelogItems:
+  - description: 'Added a help page providing search tips if a search returns no results, making it easier to troubleshoot your query.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/26154
+    category: Search
+  - description: 'In order to reduce load on the Gitserver service on installations with a large number of repositories, the search indexer only polls repositories that have been marked as changed. If you notice index staleness you can try disabling by setting the environment variable `SRC_SEARCH_INDEXER_EFFICIENT_POLLING_DISABLED` on `sourcegraph-frontend`.'
+    url: https://github.com/sourcegraph/sourcegraph/issues/27058
+    category: Search
+  - description: 'To help site admins debug issues with webhook delivery, we have added logging of incoming Batch Changes webhooks. By default, sites without encryption will log webhooks for three days and sites with encryption will not log webhooks.'
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#incoming-webhooks
+    category: Admin
+  - description: 'In order to allow writing to the `.github` directory in repositories, Batch Changes now requests the `workflow` scope on GitHub personal access tokens. If you have already configured a GitHub PAT for Batch Changes, we suggest adding this scope.'
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials
+    category: Batch Changes
+  - description: "In order to increase search speed, the search UI's repository count and the GraphQL API's `search().repositories`/`search().repositoriesCount` have changed semantics from being the set of searchable repositories to the set of repositories with matches. In a future release, we'll introduce separate fields for searchable repositories."
+    url: https://github.com/sourcegraph/sourcegraph/issues/26995
+    category: Search
+  - description: 'Version contexts and `repogroup` filters (deprecated in v3.33) have been removed in favor of search contexts.'
+    url: https://about.sourcegraph.com/blog/introducing-search-contexts/
+    category: Search
+  - description: 'Saved search notifications (deprecated in v3.31.0) have been removed in favor of code monitoring.'
+    url: https://docs.sourcegraph.com/code_monitoring
+    category: Search
+---
+
+Sourcegraph 3.34 is now available! Here are some highlights from this release:
+
+## Troubleshoot your searches with the "no results" page
+
+Searches that return no results can result in a confusing experience. When this occurs, we now display helpful tips on how to use search, examples of how to solve common query issues, and links to tutorial videos addressing common use cases.
+<img style="max-width:calc(100% - 10px)" alt="Find implementations for Go" src="https://storage.googleapis.com/sourcegraph-assets/blog/3.34/no_results_page.png">
```

Comment:
Is there some other way we can format the images? They're a little tough on the eyes at the moment with no line breaks between them and the text, and no drop shadow. I also think it'd be worthwhile to add captions so people understand what they're seeing. I tried to use the basic markdown formatting to see if that helps but that makes the images enormous 😅 @jyavorska you have helped me with rogue images in the handbook, wondering if you have ideas here?

Reply:
We can definitely change the embedded style here. It looks like bootstrap is available, so I will use some basic styling.

Reply:
Ok it should be a bit better now. I don't know where people were getting the idea to add the `style` attribute but I found it in some recent ones too and updated those to use the new `blog-images` css class. If we figure out where that is coming from, we can update the instructions for everyone to use `blog-images` instead.